immuneML/reports/encoding_reports/DesignMatrixExporter.py

```diff
@@ -29,18 +35,23 @@ class DesignMatrixExporter(EncodingReport):
     .. code-block:: yaml
 
         my_dme_report: DesignMatrixExporter
+            DesignMatrixExporter:
+                format_file: .npy/.csv/.hdf5/.npy.zip/.cvs.zip/.hdf5.zip
 
     """
-    dataset: Dataset = None
-    result_path: Path = None
-    name: str = None
+
+    def __init__(self, dataset = None, result_path = None,
+                 name: str = None, format_file: str = 'csv'):
```

Comment:
My mistake again, apologies! [and a reminder to add to the documentation]: could you remove the default value from the constructor? Default values should either not be set or be set under `immuneML/config/default_params/reports/design_matrix_exporter_params.yaml`. If they were to be set here, it would not be possible to construct the full specs after parsing that would show all the parameters used for the run, and defaults could silently change over time and the result would be different for the same specs. 🙏

Reply:
to be sure
format_file: str = 'csv'  
format_file: str = None ? 

I guess t would be better to be set at immuneML/config/default_params/reports/design_matrix_exporter_params.yaml, which should thus be a file with a single line format_file = 'csv' ?

Reply:
yes, and the line would be:

```
format_file: csv
```

Reply:
done